pkgs/applications/video/qmplay2/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, alsaLib
+, ffmpeg
+, libass
+, libcddb
+, libcdio
+, libgme
+, libpulseaudio
+, libsidplayfp
+, libva
+, libXv
+, taglib
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+let
+  pname = "qmplay2";
+  version = "20.07.04";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "zaps166";
+    repo = "QMPlay2";
+    rev = version;
+    sha256 = "sha256-sUDucxSvsdD2C2FSVrrXeHdNdrjECtJSXVr106OdHzA=";
+    fetchSubmodules = true;
```

Comment:
Can we use the system vulkan headers or override the version?

Reply:
I have added them